gender_novels/corpus.py

```diff
@@ -13,20 +14,38 @@ class Corpus(common.FileLoaderMixin):
 
     >>> from gender_novels.corpus import Corpus
     >>> c = Corpus('sample_novels')
-    >>> type(c.novels), len(c.novels)
-    (<class 'list'>, 93)
+    >>> type(c.novels), len(c)
+    (<class 'list'>, 94)
 
     >>> c.novels[0].author
     'Aanrud, Hans'
-
     """
 
     def __init__(self, corpus_name=None):
         self.corpus_name = corpus_name
         self.novels = []
         if corpus_name is not None:
+            self.relative_corpus_path = Path('corpora', self.corpus_name)
             self.novels = self._load_novels()
 
+    def __len__(self):
```

Comment:
There's already a __len__function below. Feel free to delete either.
PS: Is there a default order in which the overloaded operators should appear in a class?

Reply:
Again a merge issue (I added last week when thrrr wasn’t one). When we get lint up, a duplicated method overloaded in the same class is a bug. I have all this set up on eclipse to find it but